x-pack/plugins/task_manager/server/monitoring/capacity_estimation.ts

```diff
@@ -248,13 +248,13 @@ function getHealthStatus(
     return { status: HealthStatus.OK, reason };
   }
 
-  if (assumedAverageRecurringRequiredThroughputPerMinutePerKibana < capacityPerMinutePerKibana) {
-    const reason = `Task Manager is unhealthy, the assumedAverageRecurringRequiredThroughputPerMinutePerKibana (${assumedAverageRecurringRequiredThroughputPerMinutePerKibana}) < capacityPerMinutePerKibana (${capacityPerMinutePerKibana})`;
+  if (assumedAverageRecurringRequiredThroughputPerMinutePerKibana > capacityPerMinutePerKibana) {
```

Comment:
The logic was wrong, this should only log if we exceed `capacityPerMinutePerKibana` in this case.